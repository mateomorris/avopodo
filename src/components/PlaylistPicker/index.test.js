import React from 'react';
import { Text } from 'react-native';
import { PlaylistPicker } from './index';
import renderer from 'react-test-renderer';
import playlistIcons from 'assets/newPlaylistIcons'

jest.useFakeTimers();

it('renders PlaylistPicker correctly', () => {
  const tree = renderer
    .create(
      <PlaylistPicker 
        title={'Released when?'}
        label={'Within the past'}
        items={[
            {
                label: 'week',
                value: 'week'
            },
            {
                label: 'two weeks',
                value: 'two-weeks'
            },
            {
                label: 'month',
                value: 'month'
            },
            {
                label: 'eternity',
                value: 'eternity'
            }
        ]}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});