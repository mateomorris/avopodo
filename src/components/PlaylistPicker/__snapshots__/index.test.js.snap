// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders PlaylistPicker correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "white",
        "fontSize": 20,
        "paddingLeft": 25,
      }
    }
  >
    Released when?
  </Text>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgba(0,0,0,.3)",
        "flexDirection": "row",
        "justifyContent": "center",
        "marginBottom": 10,
        "marginTop": 10,
        "paddingLeft": 30,
        "paddingRight": 30,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "paddingRight": 2.5,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "white",
            "fontSize": 20,
            "fontWeight": "700",
          }
        }
      >
        Within the past
      </Text>
    </View>
    <View
      style={
        Object {
          "borderLeftColor": "rgba(0,0,0,.5)",
          "borderLeftWidth": 3,
          "flex": 1,
          "marginLeft": 2.5,
          "paddingLeft": 2.5,
        }
      }
    >
      <RCTPicker
        items={
          Array [
            Object {
              "label": "week",
              "textColor": undefined,
              "value": "week",
            },
            Object {
              "label": "two weeks",
              "textColor": undefined,
              "value": "two-weeks",
            },
            Object {
              "label": "month",
              "textColor": undefined,
              "value": "month",
            },
            Object {
              "label": "eternity",
              "textColor": undefined,
              "value": "eternity",
            },
          ]
        }
        onChange={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        selectedIndex={0}
        style={
          Array [
            Object {
              "height": 216,
            },
            Object {
              "color": "#eee",
              "height": 100,
              "textAlign": "left",
            },
          ]
        }
      />
    </View>
  </View>
</View>
`;
